tests/suites/0_stateless/00_dummy/00_0000_dummy_select_1.sql

```diff
@@ -1,11 +1,11 @@
 SELECT 1;
-SELECT x; -- {ErrorCode 1058}
+-- SELECT x; -- {ErrorCode 1058}
 SELECT 'a';
 SELECT NOT(1=1);
 SELECT NOT(1);
 SELECT NOT(1=1) from numbers(3);
 SELECT TRUE;
 SELECT FALSE;
 SELECT NOT(TRUE);
-SELECT 'That\'s good.';
-SELECT *; -- {ErrorCode 1065}
+SELECT 'That''s good.';
```

Comment:
Is there any problem with the new parser?

Reply:
There was before #5631 resolved.